src/configuration/routes/cache.md

```diff
@@ -85,96 +91,79 @@ cache:
   cookies: ["*"]
 ```
 
-The `Vary` header in the response is also respected. Multiple copies under
-the same cache key would be created according to the value of this header.
-For example, you can rely on the `X-Forwarded-Proto`
-[custom request header](/development/faq.md#do-you-add-custom-http-headers)
-to render content based on the request protocol (i.e. HTTP or HTTPS).
-By adding `Vary: X-Forwarded-Proto` to the response header, HTTP and HTTPS
-content would be cached under the same cache key separately.
+The `Vary` header in the response is also respected. Multiple copies under the same cache key would be created according to the value of this header. For example, you can rely on the `X-Forwarded-Proto` [custom request header](/development/faq.md#do-you-add-custom-http-headers) to render content based on the request protocol (i.e. HTTP or HTTPS). By adding `Vary: X-Forwarded-Proto` to the response header, HTTP and HTTPS content would be cached under the same cache key separately.
 
-## Cache behavior
+### Cache behavior
 
-Cache is only applied to `GET` and `HEAD` requests.
-Responses with the `Cache-Control` header set to `Private`, `No-Cache`,
-or `No-Store` are not cached. Responses with the `Set-Cookie` header set are
-also not cached.
+* Cache is only applied to `GET` and `HEAD` requests.
+* Responses with the `Cache-Control` header set to `Private`, `No-Cache`, or `No-Store` are not cached. 
+* Responses with the `Set-Cookie` header set are also not cached.
 
-## Cache duration
+### Cache duration
 
-The cache duration is decided based on the `Cache-Control` response
-header value. If no `Cache-Control` header is in the response, then the
-value of `default_ttl` key is used.
+The cache duration is decided based on the `Cache-Control` response header value. If no `Cache-Control` header is in the response, then the value of `default_ttl` key is used.
```

Comment:
We should mention somewhere that there is no flush-all command for the router.  Is this a good place for it?  If not, what would be?